src/components/common/job-indicator/index.ts

```diff
@@ -82,17 +95,20 @@ class JobIndicator extends LitElement {
     }
   `;
   
+  private context: StoreSubscriber;
+  
   constructor() {
     super();
     this.context = new StoreSubscriber(this, store);
   }
   
-  handleClick() {
+  private handleClick(): void {
     window.location.href = '/activity';
   }
   
   render() {
-    const { jobs } = this.context.store.jobsContext;
+    const jobsContext = this.context.store.jobsContext as JobsContext;
```

Comment:
We should, ideally, try to never use `as`, because this will prevent TS from actually checking `jobsContext` (the object) IS `JobsContext` (the type)

I understand why we've done it here though - because we're trying to keep this isolated. Maybe while we're migrating to TS we should come up with a pattern of `// TS-TODO` or something that needs attention later down the track when we have more of the codebase converted?